fdbserver/storageserver.actor.cpp

```diff
@@ -6592,6 +6588,7 @@ class StorageUpdater {
 	KeyRef startKey;
 	bool nowAssigned;
 	bool emptyRange;
+	UID dataMoveId;
```

Comment:
When do we use `dataMoveId`?

Reply:
They are not used in the PR.

They are used in #7340, and will be used for sharded-rocks.